tests/h/conftest.py

```diff
@@ -117,7 +117,7 @@ def restart_savepoint(session, transaction):
 @pytest.fixture(scope='session', autouse=True)
 def deform():
     """Allow tests that use deform to find our custom templates."""
-    form.init()
+    form.includeme(config=mock.Mock())
```

Comment:
This works:

```diff
diff --git a/tests/h/conftest.py b/tests/h/conftest.py
index 6cbe58e..d41d184 100644
--- a/tests/h/conftest.py
+++ b/tests/h/conftest.py
@@ -11,6 +11,7 @@ import os
 import mock
 import pytest
 
+import jinja2
 import sqlalchemy
 from pyramid import testing
 from pyramid.request import apply_request_extensions
@@ -117,7 +118,15 @@ def db_session(db_engine):
 @pytest.fixture(scope='session', autouse=True)
 def deform():
     """Use our custom Deform renderer instead of Deform's default one."""
-    form.initialize_deform_renderer(config=mock.Mock())
+    config = mock.Mock()  # Mock Pyramid Configurator object.
+
+    # Make a jinja2 environment to stand in for the app's main jinja2 env
+    # in tests that call deform.
+    env = jinja2.Environment()
+    env.globals['_'] = lambda s: s  # Our templates expect an _() function.
+    config.get_jinja2_environment.return_value = env
+
+    form.initialize_deform_renderer(config=config)
 
 
 @pytest.yield_fixture
```

Accounts views tests will now crash if our deform templates contain invalid jinja2 for example, because they actually render the templates.